packages/common/src/utils/stream-utils.ts

```diff
@@ -221,4 +221,20 @@ export class StreamUtils {
   static isAnchorCommit(commit: CeramicCommit): commit is AnchorCommit {
     return commit && (commit as AnchorCommit).proof !== undefined
   }
+
```

Comment:
Couldn't get rid of the 3 functions above and switch entirely to `CommitData` :( - open to suggestions.